client/app/components/Header/Header.scss

```diff
@@ -97,6 +97,12 @@
   }
 
   &Desktop {
+    position: fixed;
```

Comment:
With the header being fixed, you'll want to add some kind of `margin-top` value to the content of the page so that the header doesn't cover things. It should be height of the header + the existing margin or padding top value.

In addition, I think we should have a transparent background colour, I recommend using `$black-80` or `$black-90`.

Reply:
Sure, I will be working on it! I will reach out via Slack even if I have questions.